packages/core/src/external/carequality/ihe-gateway-v2/outbound/__tests__/xmls/dr-partial-success.xml

```diff
@@ -35,23 +35,20 @@
                 </rim:RegistryErrorList>
 
             </rs:RegistryResponse>
-
-            <!-- Documents available now -->
             <DocumentResponse>
```

Comment:
Why are we changing this file?

Reply:
since we changed the tests to actually use this file